### ISSUE TITLE
Byline Authors Not Sorting Correctly (defaults to alphabetical order)

### ISSUE THREAD

**Issue by atnjqt** [Automattic/Co-Authors-Plus#871]
## Overview

End users recently brought to our attention that new WP blog posts do not have correct sorting for Co Author Bylines, using both Block Editor & Classic Editor, and when editing and saving older posts (pre August 30th, 2022) the byline order is reverted to default alphabetical

### Environment:

WordPress Core **6.0.2**
PHP **8.0** running on 64bit Amazon Linux 2/3.4.1
Co Authors Plus Version **3.5.2**

**Comment by rebeccahum** [comment]
Hi, where are you seeing this? I'm unable to reproduce on wp-admin after assigning co-authors.

**Comment by atnjqt** [comment]
The plugin seems to have a mind of it's own in how to determines the author byline sorting.. I'm struggling to find a pattern which could suggest if I have misconfigured something, or if the plugin is just bizarre and buggy.

Each time we update a post's author byline ordering (i.e. to indicate who was the main author of the post), the published post itself seemingly has a fixed byline order and the editor rearranges the authors in the list... Neither of which match what I set before saving the post.

@rebeccahum if you add _many_ users, does the sorting always update on published posts as expected?
